lib/wire_client/messages/credit_transfer.rb

```diff
@@ -29,16 +29,16 @@ def build_payment_information(builder)
         builder.PmtInf do
           builder.PmtInfId(payment_information_identification(group))
           builder.PmtMtd('TRF')
+          builder.BtchBookg(group[:batch_booking])
```

Comment:
Is `BtchBookg` an XML processor? It's hard to tell from the context

Reply:
I think `BtchBookg` is just one of the weird elements that BOA demanded in the XML
so `builder.BtchBookg(group[:batch_booking])`
renders:
```
<BtchBookg>
  whatever is supposed to go here
</BtchBookg>
```